pkgs/build-support/build-bazel-package/default.nix

```diff
@@ -46,6 +46,7 @@ in stdenv.mkDerivation (fBuildAttrs // {
 
       # Patching markers to make them deterministic
       sed -i 's, -\?[0-9][0-9]*$, 1,' $bazelOut/external/\@*.marker
+      sed -i '/^ENV:TMP.*/d' $bazelOut/external/\@*.marker
```

Comment:
This can change the hash of the dependencies, I've tested `pythonPackages.tensorflow` and it still matches, but both `bazel-deps` and `bazel-watcher` don't build with the version of Bazel we have on master.